versions.tf

```diff
@@ -1,14 +1,14 @@
-# ----------------------------------------------------------------------------------------------------------------------
+# ---------------------------------------------------------------------------------------------------------------------
 # SET TERRAFORM AND PROVIDER REQUIREMENTS FOR RUNNING THIS MODULE
-# ----------------------------------------------------------------------------------------------------------------------
+# ---------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = "~> 1.0"
+  required_version = ">= 0.14, < 2.0"
```

Comment:
I think we can leave this to 
```
  required_version = "~> 1.0"
```

as we don't need to support Terraform <1.0 anymore